ecc_enclave/sgxcclib/sgxcclib.h

```diff
@@ -8,7 +8,7 @@
 #ifndef _SGXCCLIB_H_
 #define _SGXCCLIB_H_
 
-#include "types.h"
+#include "fpc-types.h"
```

Comment:
the problem with `type.h` is that the code includes the file directly -- without the path, because the path is include at link time. Same in PDO.

ok for fixing the define
ok for renaming the cpp -- but the header is still a problem because of the include in pdo